testing/httrack/APKBUILD

```diff
@@ -0,0 +1,37 @@
+# Contributor: <xmingske@gmail.com>
+# Contributor: Arch3y <arch3y@riseup.net>
+# Maintainer: Natanael Copa <ncopa@alpinelinux.org>
```

Comment:
If you're going to moved it from `unmaintained`, I'm expecting you to take up maintainership as well. :)